content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis).
+- ***Improved*** support for [Kubernetes Ingress](https://preliminary.istio.io/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and IngressClass.
+- ***Fixed*** a [bug](https://github.com/istio/istio/issues/16458) blocking external HTTPS/TCP traffic in some cases.
+- ***Removed*** most configuration flags and environment variables for the proxy. These are now read directly from the mesh configuration.
+- ***Added*** a new `proxy.istio.io/config` annotation to override proxy configuration per pod.
+- ***Added*** support for using appProtocol to select the [protocol for a port](https://preliminary.istio.io/docs/ops/configuration/traffic-management/protocol-selection/) for Kubernetes 1.18+.
+- ***Added*** the new [Workload Entry](https://preliminary.istio.io/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- ***Added*** support for configuring Gateway topology settings like number of trusted proxies deployed in front
+- Moved the proxy readiness probe to port 15021
+- TODO: add a statement around file mounted gateway removed and impact to users.
+- TODO: add a statement around networking v1alpha3 API.. will it be removed in 1.6 or announce deprecation?
+
+## Security
+
+- TODO removal of alpha API
+- SDS for mixer? Not sure we need this or its an implementation detail
+
+## Telemetry
+
+- ***Added*** Grafana dashboards to grafana.com TODO docs.
+- ***Improved*** Prometheus scraping configuration TODO docs.
+- ***Improved*** Grafana dashboards to adapt to the new Istiod deployment model.
+- ***Updated*** default Telemetry V2 configuration to avoid using host header to extract destination service name at gateway. This prevents unbound cardinality due to untrusted host header and implies that destination service labels are going to be omitted for request hits Blackhole and Passthrough at gateway.
+- ***Added*** experimental tracing options
+
```

Comment:
```suggestion
- ***Added*** Wasm extension distribution support. TODO: docs
```

@mandarjog ^^^

Reply:
Added. I'll tag Mandar when I push it up